specification/resources/databases/databases_update_installUpdate.yml

```
@@ -0,0 +1,47 @@
+operationId: databases_update_installUpdate
```

The operation ID is used by client generation tools to produce the method name. For example pydo, our Python client, will end up having a method with the redundant name `databases.update_install_update`. This might be more natural:

```suggestion
operationId: databases_install_update
```

Our linter enforces using consistent operation IDs across our API. As this would be the first PUT using `install`, you'll also need to add it here so that it passes:

https://github.com/digitalocean/openapi/blob/6fddf799ff4318258535091eed0eb7cd50bc6e6c/spectral/functions/validateOpIDNaming.js#L14

---

I've pushed an additional commit to allow this to pass the linter check: https://github.com/digitalocean/openapi/pull/909/commits/e41ef07e319a3b74505140c43a88935fcb961c63